src/main.cpp

```diff
@@ -13,14 +13,10 @@ The "main" source file with most of the boilerplate code. Includes the
 
 #include "common.hpp"
 #include "natives.hpp"
+#include <pawn-natives/NativesMain>
```

Comment:
This should be grouped with the other `<>` includes, I know it's only a small thing but it's good to keep includes grouped in some way.

Reply:
Funny thing is, I actually did that to begin with, but noticed that you hadn't done it before so I assumed you had a reason to do so and kept it the same.